src/spring-cloud/azext_spring_cloud/_params.py

```diff
@@ -281,3 +281,30 @@ def prepare_logs_argument(c):
                    arg_type=get_three_state_flag(),
                    help="Disable Application Insights.",
                    validator=validate_app_insights_parameters)
+
+    with self.argument_context('spring-cloud application-configuration-service bind') as c:
```

Comment:
```python    
with self.argument_context('spring-cloud application-configuration-service') as c:
        c.argument('service', service_name_type, validator=only_support_enterprise)
```

Reply:
Will add validator in next pr for both acs and service registry.